README.rst

```diff
@@ -47,22 +52,36 @@ website <http://gluon-nlp.mxnet.io/master/index.html>`__.
 Community
 =========
 
-For questions and comments, please visit our `forum <https://discuss.mxnet.io/>`__
-(and `Chinese version <https://discuss.gluon.ai/>`__). For bug reports, please submit Github issues.
+GluonNLP is a community that believes in sharing.
+
+For questions, comments, and bug reports, `Github issues <https://github.com/dmlc/gluon-nlp/issues>`__ is the best way to reach us.
+
+We now have a new Slack channel `here <https://apache-mxnet.slack.com/messages/CCCDM10V9>`__.
+(`register <https://join.slack.com/t/apache-mxnet/shared_invite/enQtNDIzNjY0NjQ2NjYyLTI1ZGNiOGEwOTZkMzMyOTg2ZjNkOWQyNjA5NGRhOTI5NDY4MDY0NmE1MDc2NjEzYWQ1MDZhNzU2NjE5YTMyYTA>`__).
 
 How to Contribute
 =================
 
-GluonNLP has been developed by community members. Everyone is
-more than welcome to contribute. We together can make the GluonNLP better
-and more user-friendly to more users.
+GluonNLP community welcomes contributions from anyone!
+
+There are lots of opportunities for you to become our `contributors <https://github.com/dmlc/gluon-nlp/blob/master/contributor.rst>`__:
+
+- Ask or answer questions on `GitHub issues <https://github.com/dmlc/gluon-nlp/issues>`__.
+- Propose ideas, or review proposed design ideas on `GitHub issues <https://github.com/dmlc/gluon-nlp/issues>`__.
+- Improve the `documentation <http://gluon-nlp.mxnet.io/master/index.html>`__.
+- Contribute bug reports `GitHub issues <https://github.com/dmlc/gluon-nlp/issues>`__.
+- Write new `scripts <https://github.com/dmlc/gluon-nlp/tree/master/scripts>`__ to reproduce
+  state-of-the-art results.
+- Write new `examples <https://github.com/dmlc/gluon-nlp/tree/master/docs/examples>`__.
```

Comment:
Since you mentioned what `scripts` are for, maybe also inline what `examples` are for?